client/package.json

```diff
@@ -10,7 +10,7 @@
     "@bpmn-io/align-to-origin": "^0.7.0",
     "@bpmn-io/dmn-migrate": "^0.4.3",
     "@bpmn-io/extract-process-variables": "^0.4.3",
-    "@bpmn-io/form-js": "0.0.10",
+    "@bpmn-io/form-js": "0.0.11",
```

Comment:
any plans to release 0.1.0? :P

Reply:
No.

Reply:
What would be a compeling reason?

Reply:
Agreed. There's nothing to win at the moment.